kiwi/bootloader/install/grub2.py

```diff
@@ -42,12 +42,22 @@ def post_init(self, custom_args):
                 'root device node name required for grub2 installation'
             )
 
+        self.grub2_target = 'i386-pc'
+        self.grub2_install_dev = self.device
         self.root_mount = MountManager(
             custom_args['root_device']
         )
         self.boot_mount = MountManager(
             custom_args['boot_device']
         )
+        self.modules =' '.join(Defaults.get_grub_bios_modules())
+        self.grub2_install_args = ['--skip-fs-probe']
+        if 'prep_device' in custom_args:
```

Comment:
The problem with passing custom args to a class is that you can basically specify anything. That's why I actively check for the required parts in the custom_args before proceeding. According to the changes with grub2-install the boot_device as an example became mandatory in any case. Thus I might refactor the class to pass this information as an argument instead of a custom_arg, but that's for later